### DISCUSSION TITLE
search_poi_types_descr

### DISCUSSION THREAD

**Discussion by ZeljkoBG** [osmandapp/OsmAnd#20234] [General]
_Check it out. I think something is wrong._
Key - search_poi_types_descr
Combine POI types from different categories. Tap switch to select all, tap **left side** to category selection.

Look at the application. No action is activated with "tap left side". 
There is a check box and a trigger. There is no action to the left with any of them. I guess it is confirmed by moving to the right and checking.

**Comment by dmpr0** [comment]
All works as mentioned in the provided string.

<table style="width:100%">
  <tr>
    <td><img src="https://github.com/osmandapp/OsmAnd/assets/9365308/c3bbcead-e35d-4387-9e22-94fc5f918193"  width="300" height="auto"/> </td>
    <td><img src="https://github.com/osmandapp/OsmAnd/assets/9365308/81fed17f-0520-46f3-a27f-e833f3475372"  width="300" height="auto"/> </td>
  </tr>
</table>
